src/home/home.tsx

```diff
@@ -26,6 +28,38 @@ const Home = () => {
     revalidateOnReconnect: false,
   })
 
+  const {data: configResponse} = useSWR(configUrl, fetcher, {
+    revalidateIfStale: false,
+    revalidateOnFocus: false,
+    revalidateOnReconnect: false,
+  })
+
+  const {data: encounterTypeResponse} = useSWR(encounterTypeUrl, fetcher, {
+    revalidateIfStale: false,
+    revalidateOnFocus: false,
+    revalidateOnReconnect: false,
+  })
+
+  useEffect(() => {
+    if (configResponse && !localStorage.getItem('defaultVisitType')) {
+      localStorage.setItem(
+        'defaultVisitType',
```

Comment:
can you extract `defaultVisitType` to a const variable ?

Reply:
sure @sivareddytw

Reply:
changes have been made